Bottom: c01c449ce744ef4ed5a591d6212c8668678a6dda
Top:    ed21d177d20a25a0cf9aa08cdaf0d886d4c9d64e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-23 23:58:32 +0100

action.c: Fixup warnings due to assigned but unused variables


---

diff --git a/src/action.c b/src/action.c
index 8b7e6d5..9b67924 100644
--- a/src/action.c
+++ b/src/action.c
@@ -2404,12 +2404,11 @@ static int
 ActionDisperseElements (int argc, char **argv, int x, int y)
 {
   char *function = ARG (0);
-  long minx, miny, maxx, maxy, dx, dy;
+  long minx, miny, maxy, dx, dy;
   int all = 0, bad = 0;
 
   minx = GAP;
   miny = GAP;
-  maxx = GAP;
   maxy = GAP;
 
   if (!function || !*function)
@@ -7402,8 +7401,6 @@ tempfile_name_new (char * name)
 static int
 tempfile_unlink (char * name)
 {
-  int rc;
-
 #ifdef DEBUG
     /* SDB says:  Want to keep old temp files for examiniation when debugging */
   return 0;
@@ -7413,7 +7410,7 @@ tempfile_unlink (char * name)
   int e, rc2 = 0;
   char *dname;
 
-  rc = unlink (name);
+  unlink (name);
   /* it is possible that the file was never created so it is OK if the
      unlink fails */
 
@@ -7455,7 +7452,7 @@ tempfile_unlink (char * name)
   }
 
 #else
-  rc =  unlink (name);
+  int rc = unlink (name);
 
   if (rc != 0) {
     fprintf (stderr, _("Failed to unlink \"%s\"\n"), name);
